tests/system/action/motion/test_reset_state.py

```diff
@@ -80,7 +81,8 @@ def test_reset_state_correct(self) -> None:
         assert model.get("state_id") == 76
         assert model.get("number") == "001"
         assert model.get("last_modified", 0) >= check_time
-        assert model.get("created", 0) >= check_time
+        assert model.get("workflow_timestamp", 0) >= check_time
```

Comment:
Must be equal to `last_modified`, because have been reset to None and only set, because of new state

Reply:
Okay, I changed the assert.